cache.py

```diff
@@ -1,11 +1,22 @@
 import os
 from collections import OrderedDict
 import uuid
+from dataclasses import dataclass
 
 from pytube import YouTube
 
 from urllib.parse import urlparse, parse_qs
 
+@dataclass
+class VideoInfo():
+    file_path: str
+    thumbnail: str
+    title: str
+    size_bytes: int
+
+    def __str__(self):
+        return f"VideoInfo(video_id={self.video_id}, file_path={self.file_path}, size_bytes={self.size_bytes})"
+
 class Cache():
```

Comment:
lets remove the if statement in line 35, specifically this, we dont need it anymore

```py
if (video_file_name not in os.listdir(self.file_path)):
```


and then just keep the `video.download(self.file_path)` on its own